crates/node/src/chain_spec.rs

```diff
@@ -1,5 +1,6 @@
 use std::path::PathBuf;
 
+use log::{log, Level};
```

Comment:
is this import needed? looks like it wasn't used anywhere

Reply:
Will be cleaning the PR